datafusion/src/physical_plan/file_format/parquet.rs

```diff
@@ -274,12 +274,24 @@ impl ExecutionPlan for ParquetExec {
     ) -> std::fmt::Result {
         match t {
             DisplayFormatType::Default => {
-                write!(
-                    f,
-                    "ParquetExec: limit={:?}, partitions={}",
-                    self.base_config.limit,
-                    super::FileGroupsDisplay(&self.base_config.file_groups)
-                )
+                if let Some(pre) = &self.pruning_predicate {
+                    write!(
+                        f,
+                        "ParquetExec: limit={:?}, partitions={}, pruning_predicate={}, projected_col={}",
```

Comment:
How do you like `projection` and `predicate` to shorten it? Besides, I suggest we put `project` and `predicate` before `partitions` since it might be a long list for partitions.

Reply:
Sure, I wasn't sure about the name at first👍

Reply:
I also like `projection` and `predicate`